src/box/lua/call.c

```diff
@@ -1268,6 +1269,40 @@ lbox_func_new_or_delete(struct trigger *trigger, void *event)
 	return 0;
 }
 
+static int
+lbox_rt_access_reset(struct lua_State *L)
+{
+	if (lua_gettop(L) != 0)
+		return luaL_error(L, "internal.reset_lua_call takes no "
+				  "arguments, but %d given.", lua_gettop(L));
+	rt_access_reset();
+	return 0;
+}
```

Comment:
It seems, we should do the same `uid == ADMIN` check here as in `grant_lua_call`.

Reply:
Added.